udplog/test/test_kafka.py

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) Ralph Meijer.
```

Comment:
I have removed the copyright.

On Wed, Nov 11, 2015 at 2:41 PM, Christopher Zorn <notifications@github.com>
wrote:

> In udplog/test/test_kafka.py
> <https://github.com/mochi/udplog/pull/10#discussion_r44597847>:
>
> > @@ -0,0 +1,94 @@
> > +# Copyright (c) Ralph Meijer.
>
> Your name, or remove
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mochi/udplog/pull/10/files#r44597847>.
>